src/common/ctrl_fs_util.c

```diff
@@ -135,10 +135,17 @@ int iof_ctrl_read_str(char *str, int len, const char *path)
 	if (ret != 0)
 		return ret;
 
+	buf = malloc(IOF_CTRL_MAX_LEN);
```

Comment:
why is this using malloc?

Reply:
probably because this is a library utility